lib/guardian/plug/ensure_authenticated.ex

```diff
@@ -27,22 +30,32 @@ defmodule Guardian.Plug.EnsureAuthenticated do
     opts = Enum.into(opts, %{})
     handler = build_handler_tuple(opts)
 
-    claims_to_check = Map.drop(opts, [:on_failure, :key, :handler])
+    claims_to_check = Map.drop(opts, [:on_failure, :key, :handler, :except])
     %{
       handler: handler,
       key: Map.get(opts, :key, :default),
-      claims: Guardian.Utils.stringify_keys(claims_to_check)
+      claims: Guardian.Utils.stringify_keys(claims_to_check),
+      except: Map.get(opts, :except) || []
     }
   end
 
   @doc false
   def call(conn, opts) do
     key = Map.get(opts, :key, :default)
 
-    case Guardian.Plug.claims(conn, key) do
-      {:ok, claims} -> conn |> check_claims(opts, claims)
-      {:error, reason} -> handle_error(conn, {:error, reason}, opts)
-      _ -> handle_error(conn, {:error, :no_session}, opts)
+    phoenix_action = conn
+      |> Map.get(:private)
+      |> Map.get(:phoenix_action)
+    allowed_actions = Map.get(opts, :except)
```

Comment:
`allowed_actions` feels strange to me here.  Could we rename this to something akin to `exempted_actions`?